src/main/cpp/t04_swap.cpp

```diff
@@ -25,6 +25,26 @@
 using namespace std;
 
 int t04_swap() {
-    
+    int N=0, M=0, x1=0, y2=0, swp=0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'swp' is assigned a value that is never used.](https://app.codacy.com/manual/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4734892)